bin/alluxio-start.sh

```diff
@@ -298,11 +298,15 @@ start_worker() {
         fi
 
         # recursively delete all (including hidden) files of the ramdisk
-        find "${dir}" -mindepth 1 -delete
+        # - avoiding deleting the directory itself to avoid permission
+        #   changes/requirements on the parent directory
+        shopt -s dotglob
+        rm -rf "${dir}/"*
+        shopt -u dotglob
```

Comment:
The `*` is outside the `"${dir}/"` double-quotes so the glob actually gets evaluated.